OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -155,14 +152,9 @@ void CacheProduceables(Actor playerActor)
 			{
 				var bi = a.Traits.Get<BuildableInfo>();
 
-				// Can our race build this by satisfying normal prerequisites?
-				var buildable = !Info.RequireOwner || bi.Owner.Contains(Race);
-
 				// Checks if Prerequisites want to hide the Actor from buildQueue if they are false
```

Comment:
There is no checking done here any more, so this comment should probably go as well.